docs/data/joy/components/input/input.md

```diff
@@ -166,6 +166,24 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "InputVariables.js", "hideToolbar": true}}
 
+## Integration with 3rd party libraries
+
+To integrate Input with 3rd party libraries, create a wrapper component to map props with 3rd party component API.
+
+Then use the wrapper component as a value to `slotProps.input.component` of Joy UI Input.
+
+### React imask
+
+[react-imask](https://github.com/uNmAnNeR/imaskjs) provides `IMaskInput` component for creating a wrapper component.
+
+{{"demo": "InputReactImask.js"}}
+
+### React number format
+
+[react-number-format](https://github.com/s-yadav/react-number-format) provides `NumericFormat` component for creating a wrapper component.
```

Comment:
```suggestion
[react-number-format](https://github.com/s-yadav/react-number-format) provides the `NumericFormat` component enabling the formatting of texts that meet a specific number or string pattern.
```

Trying to convey what is the value behind this library in a more descriptive way. Pulled some words from their read.me.